packages/ncids-css/packages/nci-guide-card/internal/_nci-with-image-and-description.scss

```diff
@@ -1,33 +1,43 @@
 @use "uswds-core" as *;
 
 .nci-guide-card--with-image-and-description {
-	.nci-guide-card__image {
-		img {
-			@include u-display('block');
-			@include u-maxw('full');
-			@include u-height('auto');
+	@include u-radius('md');
+	.nci-guide-card {
```

Comment:
This seems wrong here. The `nci-guide-card--with-image-and-description` is the variant of the `nci-guide-card`. Shouldn't this be `&.nci-guide-card`?